packages/core/EventManager.ts

```diff
@@ -44,6 +45,13 @@ interface HasId {
   id: number;
 }
 
+// The options should have the slug of the apps the option is enabled for
+interface AppOptions {
```

Comment:
Decided to take this approach in case we have more apps in the future that we need to pass options down into.